core/src/main/java/org/bitcoinj/wallet/DeterministicKeyChain.java

```diff
@@ -1227,7 +1253,7 @@ public DeterministicSeed getSeed() {
         ImmutableList.Builder<DeterministicKey> keys = ImmutableList.builder();
         for (ECKey key : getKeys(true)) {
             DeterministicKey dKey = (DeterministicKey) key;
-            if (dKey.getPath().size() > 2) {
+            if (dKey.getPath().size() == getAccountPath().size() + 2) {
```

Comment:
Hm is this change right? It's gone from everything below a certain path to just that single level.